rbac/api/common/exception_handler.py

```diff
@@ -87,6 +90,18 @@ def _generate_error_data_payload_response(detail: str, context, http_status_code
     return data
 
 
+def problem_json_exception_handler(exc, context):
+    response = exception_handler(exc, context)
+    if isinstance(exc, exceptions.ValidationError):
```

Comment:
This ensures we not only handle validation errors from the serializer, but also the model.